### DISCUSSION TITLE
Error: package or namespace load failed for ‘clusterProfiler’ in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]):  不存在叫‘HPO.db’这个名字的程辑包

### DISCUSSION THREAD

**Discussion by ymy4984** [YuLab-SMU/clusterProfiler#619] [Q&A]
At 30 Aug, 2023, I can library(clusterProfiler). However, at 2nd Sep, 2023, i cannot library(clusterProfiler).

Solution #1: i deleted clusterProfiler and re-installed it. But it showed Error: package or namespace load failed for ‘clusterProfiler’ in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]):
 不存在叫‘HPO.db’这个名字的程辑包

Next, i tried to BiocManager::install("HPO.db") but it failed, showing "error: database disk image is malformed".

And i log in HPO database website (https://hpo.jax.org/app/data/ontology), showing the database is upgrading.

My version: Bioconductor version 3.18 (BiocManager 1.30.22), R 4.3.1 (2023-06-16 ucrt)

**Comment by huerqiang** [comment]
This is an error in AnnotationHub package, not HPO.db or clusterProfiler: https://support.bioconductor.org/p/121979/. Or try to download and install the local version of HPO.db: https://github.com/huerqiang/HPO.db_MPO.db.
